pwsh_module/command.tests.ps1

```diff
@@ -0,0 +1,242 @@
+BeforeAll {
```

Comment:
Should we run these as part of CI? We can set up a GH Action that only runs them if the pwsh module is touched by a PR.

Reply:
Yes we should, but I didn't think about adding it in this PR. Do you think it should go in the existing `windows.yaml` file or a new file?

Reply:
I added another file since I realized we need to trigger based on path and the existing ones are for all pull requests or pushes. Now I have to figure out why pester is failing in github but passes locally

Reply:
Pester now works in CI because I have the scoping corrected